app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,22 @@
+<% provide:title,(t ".about") %>
+<h1><%= t ".about" %></h1>
+<p>
+	<%= t ".the" %>
+	<a href="http://www.railstutorial.org/">
+		<em>
+			<%= t ".about_ruby" %>
+		</em>
+	</a> <%= t ".is_a" %>
+	<a href="http://www.railstutorial.org/book">
+		<%= t ".book" %>
+	</a>
+	<%= t ".and" %>
+	<a href="http://screencasts.railstutorial.org/">
+		<%= t ".screencast_series" %>
+	</a>
+	<%= t ".to_teach" %>
+	<a href="http://rubyonrails.org/">
+		<%= t ".ruby_on_rails" %>
+	</a>.
+	<%= t ".this_is_the" %>
+</p>
```

Comment:
Toàn bộ file này em đang để khoảng trắng code rất ko tốt, rất lộn xộn và xấu, em xem lại nhé